src/ArrayCache.php

```diff
@@ -88,7 +88,7 @@ public function deleteMultiple($keys): bool
         $keys = $this->iterableToArray($keys);
         $this->validateKeys($keys);
         foreach ($keys as $key) {
-            $this->delete($key);
+            $this->delete((string)$key);
```

Comment:
This is unnecessary - `validateKeys()` already ensures that all keys are strings.